test/example-types/dateHistogram.js

```diff
@@ -56,7 +56,7 @@ describe('dateHistogram', () => {
             min: { $min: '$metrics.usersCount' },
             sum: { $sum: '$metrics.usersCount' },
             count: { $sum: 1 },
-            cardinality: { $addToSet: '$metrics.usersCount' },
+            cardinality: { $sum: '$metrics.usersCount' },
```

Comment:
same as with `dateHistogram` itself -- I couldn't check in my actual changes because this test was failing on master.